packages/mantine/package.json

```diff
@@ -0,0 +1,89 @@
+{
+    "name": "@coveord/plasma-mantine",
+    "version": "0.0.0",
+    "description": "A Plasma flavoured Mantine theme",
+    "keywords": [
+        "plasma",
+        "mantine",
+        "components",
+        "coveo"
+    ],
+    "license": "ISC",
+    "author": "",
+    "exports": {
+        ".": {
+            "types": "./dist/definitions/index.d.ts",
+            "import": "./dist/esm/index.js",
+            "require": "./dist/cjs/index.js"
+        }
+    },
+    "main": "./dist/cjs/index.js",
+    "module": "dist/esm/index.js",
+    "types": "./dist/definitions/index.d.ts",
+    "scripts": {
+        "build": "node ../../scripts/build",
+        "clean": "rimraf dist",
+        "start": "node ../../scripts/start",
+        "test": "jest --maxWorkers=65%"
+    },
+    "dependencies": {
+        "@coveord/plasma-react-icons": "workspace:*",
+        "@coveord/plasma-tokens": "workspace:*",
+        "@emotion/react": "11.10.0",
+        "@monaco-editor/react": "4.4.5",
+        "@swc/helpers": "0.4.11",
+        "@tanstack/react-table": "8.5.11",
+        "@tanstack/table-core": "8.5.11",
+        "dayjs": "1.11.3",
+        "lodash.defaultsdeep": "4.6.1",
+        "monaco-editor": "0.34.0",
+        "react-beautiful-dnd": "13.1.0",
+        "react-redux": "7.2.8",
+        "redux": "4.2.0"
+    },
+    "devDependencies": {
+        "@emotion/react": "11.10.0",
+        "@mantine/core": "5.6.2",
+        "@mantine/dates": "5.6.2",
+        "@mantine/form": "5.6.2",
+        "@mantine/hooks": "5.6.2",
+        "@mantine/modals": "5.6.2",
```

Comment:
Shouldn't this be dependencies?

Reply:
We decided to put them in the peerDeps + devDeps to allow users of plasma-mantine to have more control over the version of mantine they want to use. Just like we do for react and react-dom